frontend/common/services/useRolePermission.ts

```diff
@@ -26,48 +26,27 @@ export const rolePermissionService = service
 
       getRoleEnvironmentPermissions: builder.query<
         Res['rolePermission'],
-        Req['getRolePermission']
+        Req['getRolePermissionEnvironment']
       >({
-        providesTags: (res) => [{ id: res?.id, type: 'rolePermission' }],
-        query: (query: Req['getRolePermission']) => ({
+        query: (query: Req['getRolePermissionEnvironment']) => ({
           url: `organisations/${query.organisation_id}/roles/${query.role_id}/environments-permissions/?environment=${query.env_id}`,
         }),
       }),
+
       getRoleOrganisationPermissions: builder.query<
         Res['rolePermission'],
-        Req['getRolePermission']
+        Req['getRolePermissionOrganisation']
       >({
-        providesTags: (res) => [{ id: res?.id, type: 'rolePermission' }],
-        query: (query: Req['getRolePermission']) => ({
+        query: (query: Req['getRolePermissionOrganisation']) => ({
           url: `organisations/${query.organisation_id}/roles/${query.role_id}/organisation-permissions/`,
         }),
       }),
-      getRoleProjectPermissions: builder.query<
```

Comment:
There were duplicates here, getRoleProjectPermissions and getRolesProjectPermissions